resources/zib/zib-AdministrationAgreement.xml

```diff
@@ -0,0 +1,282 @@
+        <map value="NL-CM:9.8.22660" />
+        <comment value="PeriodOfUse" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.extension:periodOfUseDuration">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="periodOfUseDuration" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval-Duration" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:periodOfUseDuration.value[x]">
+      <path value="MedicationDispense.extension.value[x]" />
+      <short value="PeriodOfUse / Duration" />
+      <definition value="**Duration**: The intended duration of use. E.g. 5 days or 8 weeks. It is not allowed to indicate the duration in months, because different months have a variable duration in days." />
+      <alias value="Gebruiksperiode" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.22660" />
+        <comment value="PeriodOfUse" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.modifierExtension">
+      <path value="MedicationDispense.modifierExtension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationDispense.modifierExtension:stopType">
+      <path value="MedicationDispense.modifierExtension" />
+      <sliceName value="stopType" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-StopType" />
+      </type>
+    </element>
+    <element id="MedicationDispense.modifierExtension:stopType.value[x]">
+      <path value="MedicationDispense.modifierExtension.value[x]" />
+      <short value="AdministrationAgreementStopType" />
+      <alias value="ToedieningsafspraakStopType" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.22498" />
+        <comment value="AdministrationAgreementStopType" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.modifierExtension:repeatPeriodCyclicalSchedule">
+      <path value="MedicationDispense.modifierExtension" />
+      <sliceName value="repeatPeriodCyclicalSchedule" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-InstructionsForUse.RepeatPeriodCyclicalSchedule" />
+      </type>
+    </element>
+    <element id="MedicationDispense.status">
+      <path value="MedicationDispense.status" />
+      <short value="CanceledIndicator" />
+      <definition value="A code specifying the state of the set of dispense events. In the event of an error correction, the value of this element shall be 'entered-in-error'." />

```

Comment:
This guidance is not clear enough. Does CanceledIndicator == true imply that `.status` = _entered-in-error_ or does it work in one way only? And if does work in one way, should CanceledIndicator not guide status to `cancelled`? And what if CanceldIndicator is false or absent?

Reply:
The `comment` states: 'When `status` = _entered-in-error_ CanceledIndicator is _true_.'. I would say this is the only condition that CanceledIndiator is true. If `status` is another code, CanceledIndicator may be implied false or absent? Shall I add that to the guidance?

The definition of CanceldIndicator: "In the event of an error correction, this indicator is to be put on for the incorrect agreement."
The definitions of _cancelled_ and _entered-in-error_:
- cancelled - The dispensed product was not and will never be picked up by the patient.
- entered-in-error - The dispense was entered in error and therefore nullified.

Based on the definitions of the concept, I believe the entered-in-error is a proper match.

Reply:
I improved the mapping guidance on this element according to our discussion.